### PR TITLE
Realm Web: Fixed TypeScript bundle

### PR DESCRIPTION
**Pull request by kraenhansen**
## What, How & Why?

This closes an issue where the Realm Web typescript type bundle would include a triple-slash reference to "realm".

Note: This should be rebased once the current base is merged into `v10`.
Note: This has already been released as Realm Web v0.3.0.

## ☑️ ToDos
* [x] 📝 Changelog entry
* [x] 🚦 Tests
* [x] 📝 Public documentation PR created or is not necessary
* [x] 💥 `Breaking` label has been applied or is not necessary

*If this PR adds or changes public API's:*
* [x] typescript definitions file is updated
* [ ] jsdoc files updated
* [x] Chrome debug API is updated if API is available on React Native

### CODE REVIEW

File: packages/realm-web/README.md [hunk 1]
```diff
@@ -4,15 +4,13 @@ Accessing MongoDB Realm from a web-browser.
...
-
 ```
-npm install https://static.realm.io/downloads/realm-web/realm-web-0.1.0.tgz
+npm install realm-web --save
 ```
 
 ## Caveats / limitations
 
-As this is a *very early* preview release, it comes with a few caveats:
+As this is a beta release, it comes with a few caveats:
```
Comment (kneth) [inline packages/realm-web/README.md:13]: :smile:

**Comment by kraenhansen** [comment]
I added code to fail the integration tests if errors or warnings are printed in the console of the browser.

File: packages/realm-web/README.md [hunk 2]
```diff
@@ -4,15 +4,13 @@ Accessing MongoDB Realm from a web-browser.
 
 ## Installation
 
-In its current form, the package has not been published to NPM, instead its distributed using a direct HTTP URL:
-
 ```
-npm install https://static.realm.io/downloads/realm-web/realm-web-0.1.0.tgz
+npm install realm-web --save
```
Comment (steffenagger) [inline packages/realm-web/README.md:8]: `--save` has been the default behaviour since `v5.0.0`.

Comment (kraenhansen) [reply packages/realm-web/README.md:8]: Interesting ...

**Pull request merged**